flatcar-postinst

```diff
@@ -294,34 +295,40 @@ if [[ -z "${LDSO}" ]]; then
 fi
 LIBS="${LDSO%/*}"
 
-# Use torcx binary from next-OS image to ensure compatibility
-TORCX_BIN=""
-for bindir in bin/old_bins bin sbin lib/coreos; do
-    if [[ -x "${INSTALL_MNT}/${bindir}/torcx" ]]; then
-        TORCX_BIN="${INSTALL_MNT}/${bindir}/torcx"
-        break
+# Flatcar major releases > 3760 do not ship torcx (these use sysext)
+FINAL_TORCX_RELEASE=3760
```

Comment:
Why don't we just drop the code? We don't expect to backport this change to older channels.

Reply:
We might end up in a situation where we need to ship a critical fix for update engine for an older release, e.g. Stable or LTS. I'm concerned that in the heat of that moment (shipping critical fixes can be quite involving) we'll forget about the missing torcx logic.

Reply:
For the record if someone is looking why this was removed instead of being hidden behind a version check: for critical fixes to older releases we can create a backport branch for the release that will only contain a commit with a critical fix, without the additional baggage.